kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/RuntimeDataResource.java

```diff
@@ -528,14 +528,19 @@ public Response getTaskByWorkItemId(@Context HttpHeaders headers,
     @Path(TASK_GET_URI)
     @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
     public Response getTaskById(@Context HttpHeaders headers, 
-            @ApiParam(value = "task id to load task instance", required = true) @PathParam(TASK_INSTANCE_ID) Long taskId) {
+            @ApiParam(value = "task id to load task instance", required = true) @PathParam(TASK_INSTANCE_ID) Long taskId,
+            @ApiParam(value = "optional include SLA data - defaults to false", required = false) @QueryParam("includeSLA") @DefaultValue("false") boolean includeSLA) {
         Variant v = getVariant(headers);
         // no container id available so only used to transfer conversation id if given by client
         Header conversationIdHeader = buildConversationIdHeader("", context, headers);
 
         TaskInstance userTaskDesc = null;
         try {
-            userTaskDesc = runtimeDataServiceBase.getTaskById(taskId);
+            if(includeSLA){
+                userTaskDesc = runtimeDataServiceBase.getTaskById(taskId);
+            } else {
+               userTaskDesc = runtimeDataServiceBase.getTaskByIdWithSLA(taskId);
+            }
```

Comment:
I think this should be the other way around.

Reply:
yes, updated.